src/types_10730.ts

```diff
@@ -0,0 +1,14 @@
+import type {
+  RegistryTypes,
+} from "@polkadot/types/types"
+import { types10720 } from "./types_10720";
+
+export const types10730: RegistryTypes = {
+  ...types10720,
+  StakingRates: {
```

Comment:
Can you maybe add a comment to mark the type section relative to the staking runtime API?

Reply:
[5c65833](https://github.com/KILTprotocol/type-definitions/pull/35/commits/5c65833d1284157689928afb1ceba5cf6873a271)